utils/oc.py

```diff
@@ -256,6 +258,10 @@ def get_api_resources(self):
         results = self._run(cmd).decode('utf-8').split('\n')
         return [r.split()[-1] for r in results]
 
+    def get_oc_version(self):
```

Comment:
i'm not seeing usage of this function. am i missing it?
and just a nit - can we rename it to `version`? we are in the context of `oc`, and `version` is not `get`.

Reply:
Sure thing!